resources/geocoding/en/52.txt

```diff
@@ -1,819 +1,420 @@
-5255|Mexico City
-52222|Amozoc
-52222|Puebla
-52223|Magdalena Tetela
-52224|Atoyatempan
+# Copyright (C) 2011 The Libphonenumber Authors
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Generated from:
+# http://www.itu.int/oth/T020200008A/en
+# http://numeracion.ift.org.mx/numeracion.exe/w_edoymun
+#
+# For determine the city's geocoding:
+#    http://numeracion.ift.org.mx/numeracion.exe/info_asl?cld=<lada>
+#   This page shows "Ciudad con mayor cantidad de numeración asignada"
+
+
+52222|Heroica Puebla De Zaragoza
+52223|Tepeaca
+52224|Tecali De Herrera
+52225|Tlapacoyan
 52226|Altotonga
-52226|Jalacingo
 52227|Huejotzingo
-52228|Jalapa
-52232|La Vigueta
+52228|Xalapa-enriquez
+52229|Veracruz
+52231|Teziutlan
 52232|Martinez De La Torre
```

Comment:
Title-casing is I presume done automatically by you? I think we need a search-replace to fix "De" and "La" to  "de" and "la" where they aren't at the start of a name, to make it look more user-friendly.